dts/bindings/display/backlight.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024, Martin Kiepfer
+# SPDX-License-Identifier: Apache-2.0
+
+include: [base.yaml]
+
+properties:
+  init-level:
+    type: int
+    default: 80
+    description: Initial value of the Backlight.
+
+  disable-init-on-boot:
+    type: boolean
+    description: |
+      Disable the initial value during bootup of the device. By default the
+      init-level is applied on boot. This option can prevent from this default
+      behaviour. Backlight needs to be enabled manually later in this case.
+
+  label:
```

Comment:
label is already included in base.yaml. Is there a need to redefine it?

Reply:
yes, it has to be redefined, for two reasons. 
1) label is deprecated in base.yaml (see #80863)
2) according to DT spec, eaach device binding should describe what does the label mean for that device